drivers/python/requirements-test.txt

```diff
@@ -0,0 +1,4 @@
+PyOpenSSL
+PyYAML
```

Comment:
The `PyYAML` requirement is only on the test system, and there are plans to remove it (#4787)